### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by SULDomiYT**
## Type of change

- [ ] Add new directory for a server
- [ ] Update directory of a server
- [ ] Documentation Update

## Checklist

- [ ] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [ ] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [ ] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/sulentia/manifest.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "de"
+  ],
+  "social": {
+    "web": "https://sulentia.net",
+    "web_support": "https://forum.sulentia.net",
+    "twitter": "NetzwerkSul",
+    "discord": "https://dc.sulentia.net",
+    "teamspeak": "sulentia.net",
+    "youtube": "https://www.youtube.com/channel/UCeAYM9xJTEZLj8zm0IQdOnw"
+  }
```
Comment (JakobBruening) [inline minecraft_servers/sulentia/manifest.json:18]: Add comma for valid syntax.

File: minecraft_servers/sulentia/manifest.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  "discord": {
+    "server_id": 841711753876078622,
+    "rename_to_minecraft_name": true
```
[the remark this answers is not in the document]
Comment (SULDomiYT) [reply minecraft_servers/sulentia/manifest.json]: the

**Review by JakobBruening — CHANGES_REQUESTED** [review]
Please compress the images.

**Review by JakobBruening — APPROVED** [review]
Thank you! :)

**Pull request merged**